templates/deployments/appname/app.jsonnet.tpl

```diff
@@ -292,11 +292,19 @@ local all = {
 								send_distribution_buckets: true,
 							},
 						],
-						{{- if not (empty (stencil.Arg "kubernetes.groups")) }}
-						['ad.datadoghq.com/' + k8sMetricsName + '.check_names']: '["openmetrics"]',
-						['ad.datadoghq.com/' + k8sMetricsName + '.init_configs']: '[{}]',
-						['ad.datadoghq.com/' + k8sMetricsName + '.instances']: std.manifestJsonEx(self.k8s_datadog_prom_instances_, '  '),
+            {{- else }}
+						['ad.datadoghq.com/' +  app.name + '.check_names']: '["openmetrics","openmetrics"]',
```

Comment:
Can we add a comment here about why the value is duplicated. Just worried someone will accidentally remove this later

Reply:
done